html/semantics/forms/constraints/form-validation-validity-patternMismatch.html

```diff
@@ -18,8 +18,11 @@
         {conditions: {pattern: "[A-Z]+", value: ""}, expected: false, name: "[target] The value attibute is empty string"},
         {conditions: {pattern: "[A-Z]{1}", value: "A"}, expected: false, name: "[target] The value attribute matches the pattern attribute"},
         {conditions: {pattern: "[A-Z]+", value: "\u0041\u0042\u0043"}, expected: false, name: "[target] The value(ABC) in unicode attribute matches the pattern attribute"},
-        {conditions: {pattern: "[a-z]{3,}", value: "ABCD"}, expected: true, name: "[target] The value attribute mismatches the pattern attribute"},
-        {conditions: {pattern: "[A-Z]+", value: "ABC123"}, expected: true, name: "[target] The value attribute mismatches the pattern attribute even when a subset matches"}
+        {conditions: {pattern: "[a-z]{3,}", value: "ABCD"}, expected: true, acceptableValue: "abcd", name: "[target] The value attribute mismatches the pattern attribute"},
```

Comment:
I feel it's difficult to imagine what is tested by `acceptableValue`.

I'd like to do:
 - Not update validator.js
 - Add new testElements item like:
```
{
  tag: "input",
  types: ["email"],
  testData: [
    {conditions: {pattern: "[a-z]{3,}", multiple: true, value: "ABCD, abcd"}, expected: ....},
```

Reply:
Hmm, that looks much better. Updated, thanks!